app/assets/stylesheets/components/_option-select.scss

```diff
@@ -103,4 +103,44 @@
       }
     }
   }
+
+  // This is a temporary overwrite for checkboxes
+  // These styles should be removed in favour of small checkboxes from the Design System
+  .govuk-checkboxes__item {
+    min-height: 26px;
+    margin-bottom: 10px;
+    padding: 0 0 0 30px;
+  }
+
+  .govuk-checkboxes__item:last-child,
+  .govuk-checkboxes__item:last-of-type {
+    margin-bottom: 0;
+  }
+
+  .govuk-checkboxes__input {
+    top: 5px;
+    left: 0;
+    width: 26px;
+    height: 26px;
+  }
+
+  .govuk-checkboxes__label {
+    padding: 8px 5px 2px;
+    font-size: 16px;
```

Comment:
Should we be using a mixin for font sizes or is that not appropriate here?

Reply:
Yep, all values, including other spacing values would ideally use variables and mixins, but I didn't bother since there's no precedent in this repo and this is a 'temporary' code.